arangodb-net-standard/GraphApi/GraphApiClient.cs

```diff
@@ -136,5 +137,31 @@ public async Task<GetGraphEdgeCollectionsResponse> GetGraphEdgeCollectionsAsync(
                 throw await GetApiErrorException(response);
             }
         }
+
+        /// <summary>
+        /// Adds an additional edge definition to the graph.
+        /// This edge definition has to contain a collection and an array of
+        /// each from and to vertex collections.An edge definition can only
+        /// be added if this definition is either not used in any other graph, or
+        /// it is used with exactly the same definition. It is not possible to
+        /// store a definition “e” from “v1” to “v2” in the one graph, and “e”
+        /// from “v2” to “v1” in the other graph.
+        /// </summary>
+        /// <param name="graphName"></param>
+        /// <param name="body"></param>
+        /// <returns></returns>
+        public async Task<PostGraphEdgeResponse> PostGraphEdgeDefinitionAsync(string graphName, EdgeDefinition body)
```

Comment:
Should body be of type `PostGraphEdgeBody` instead of `EdgeDefinition`?

Reply:
Well, the class already contains all the properties needed. So I saw little benefit from repeating code

Reply:
But yeah, may aswell use PostGraphEdgeBody

Reply:
It could be good to reuse some models when we are sure new properties are unlikely to appear yes. Maybe we can have a look at what our models look like once we finish v1.